platforms/enterprise/enterprise-plugin-performance/build.gradle.kts

```diff
@@ -8,12 +8,23 @@ description = """Performance tests for the build scan plugin
     | """.trimMargin()
 
 dependencies {
-    testFixturesApi(project(":internal-performance-testing"))
-    testFixturesApi(libs.commonsIo)
+    performanceTestImplementation(project(":base-services"))
```

Comment:
🤔 There is a risk here that the performance tests break, since we don't have this in our pipeline, but in the DV pipeline. We can go ahead merging the PR and see if it breaks, or you can run (part of) the performance test and see if it still runs, though it isn't trivial.

Reply:
Looks like they passed!